Mikado/Mikado/Model/TripCell.swift

```diff
@@ -5,30 +5,53 @@
 //  Created by Aziz Zaynutdinov on 2/10/19.
 //  Copyright © 2019 Aziz Zaynutdinov. All rights reserved.
 //
-
 import UIKit
 
 class TripCell: UITableViewCell {
 
+    @IBOutlet weak var cityImage: UIImageView!
+    @IBOutlet weak var tripName: UILabel!
```

Comment:
Small comment, typically name variables based on what the type is, like cityImageView and tripNameLabel or nameLabel. tripName might be ambiguous as it may imply that it is of type String instead of UILabel. Its up to you if you want to change it or not :)